distributed/tests/test_preload.py

```diff
@@ -121,3 +123,23 @@ async def test_preload_import_time(cleanup):
         from distributed.comm.registry import backends
 
         del backends["foo"]
+
+
+@pytest.mark.asyncio
+async def test_web_preload(cleanup):
+    class MyHandler(web.RequestHandler):
+        def get(self):
+            self.write(
+                """
+def dask_setup(dask_server):
+    dask_server.foo = 1
+""".strip()
+            )
+
+    app = web.Application([(r"/preload", MyHandler)])
+    server = app.listen(12345)
+    try:
+        async with Scheduler(preload=["http://localhost:12345/preload"]) as s:
+            assert s.foo == 1
+    finally:
+        server.stop()
```

Comment:
This was the primary motivation for this PR

Reply:
I actually don't see the motivation for this, can you expand? `exec`'ing code that is loaded from an http response seems like a risky thing to enable users to do - without proper security steps to prevent MITM attacks, this could load and exec a malicious script.